src/Equations/ConfinementTime.cpp

```diff
@@ -55,11 +92,160 @@ real_t ConfinementTime::EvaluateParallelConfinementTime(len_t ir) {
 /**
  * Evaluates the perpendicular confinement time.
  */
-real_t ConfinementTime::EvaluatePerpendicularConfinementTime(len_t ir) {
-    real_t a      = radials->GetMinorRadius();
-    real_t B      = radials->GetMagneticField();
-    real_t T_cold = unknowns->GetUnknownData(id_Tcold)[ir];
-    return T_cold/(8*a*a*B);
+
+//Possible improvement using a lambda function
+//Return a power list containing the coeffcients for each law stored in a map in a .hpp file
+const PowerList ConfinementTime::GetCoeff(STREAM::OptionConstants::Conf_Time_type _type) const 
+{
+	return COEFFICIENTS::LawCoefficients.at(_type);
+}
+
+const PowerList ConfinementTime::GetCoeff() const //Return a power list containing the coeffcients for each law stored in a map in a .hpp file
+{
+	return COEFFICIENTS::LawCoefficients.at(type);
+}
+
+//Computation od the ohmic power
+real_t ConfinementTime::GetOhmicPower(len_t ir) const
+{
+	real_t V = eqsys->GetPlasmaVolume()->GetPlasmaVolume();
+	real_t E_field = unknowns->GetUnknownData(id_Efield)[ir];	
+	real_t sigma = eqsys->GetREFluid()->GetElectricConductivity(ir);
+	real_t R0 = radials->GetMajorRadius();
+	
+	return E_field * E_field * sigma * V / R0; // Need to normalize by R0 
```

Comment:
** need to normalize volume by R0 to be physically a volume (m^3)

Reply:
Done